integration-tests/src/test/java/org/zowe/apiml/integration/authentication/providers/LogoutTest.java

```diff
@@ -32,7 +32,7 @@
 class LogoutTest implements TestWithStartedInstances {
 
     protected static String[] logoutUrlsSource() {
-        return new String[]{SecurityUtils.getGatewayLogoutUrl(), SecurityUtils.getGatewayLogoutUrlOldPath()};
+        return new String[]{SecurityUtils.getGatewayLogoutUrl()};
```

Comment:
Does it make sense to have parametrized method then?

Reply:
Removed